modules/landing_zone/scripts/config.js

```diff
@@ -17,6 +17,10 @@ async function checkEnvironmentVars() {
     return Promise.reject(Error('ERROR: PROVIDERS variable is empty. Aborting...'));
   }
 
+  if (!backend) {
```

Comment:
I believe we agreed that backend variable is optional ... am I missing something?